alternator/executor.cc

```diff
@@ -2644,7 +2644,7 @@ void executor::describe_single_item(const cql3::selection::selection& selection,
     const std::vector<managed_bytes_opt>& result_row,
     const std::optional<attrs_to_get>& attrs_to_get,
     rjson::value& item,
-    consumed_capacity_counter* consumed_capacity_collector,
+    uint64_t* consumed_capacity_collector,
```

Comment:
‎I find this change surprising, because you could have always used a bare uint64_t instead of a sophisticated "consumed_capacity_counter" but you deliberately wanted it to be a class. And here you are (partially) reverting to just use a counter.
I think this change is fine, but please consider if you should change the variable's name. It's no longer fits the type it implies. Maybe you should call it "total_byte_counter" or something like that?

Reply:
This change is just for the method; while it was used for the counter (which still exists) it could change it directly now that there are other options, I've changed it to uint64_t pointer